superset/assets/javascripts/components/TooltipWrapper.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+import { Tooltip, OverlayTrigger } from 'react-bootstrap';
+import { slugify } from '../modules/utils';
+
+const propTypes = {
+  label: PropTypes.string.isRequired,
+  tooltip: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+  placement: PropTypes.string,
+};
+
+const defaultProps = {
+  placement: 'top',
+}
+
+export default function TooltipWrapper({ label, tooltip, children, placement}) {
+  console.log('children', children)
```

Comment:
yep. looks like my linter stopped working in sublime for some reason, so i missed this. thanks for the :eyes: